### PR TITLE
Add cropping example

### PR DESCRIPTION
**Pull request by thecre8tor**
Adds an example of the `Canvas::crop` method part of #1

### CODE REVIEW

File: examples/canvas.rs [hunk 1]
```diff
@@ -46,10 +46,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         canvas.place_at(Point::zero()).draw(&mut display)?;
 
         // prepare for drawing
-        let drawing_size = Size {
-            width: 100,
-            height: 100,
-        };
+        let drawing_size = Size {width: 100,height: 100};
```
Comment (elpiel) [inline examples/canvas.rs]: You can use `cargo fmt` to format your code, this shouldn't have been changed as it was already formatted.

File: examples/cropped_text.rs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    canvas.place_at(canvas.center() / 3).draw(&mut display)?;
+
+    // let thick_stroke = PrimitiveStyle::with_stroke(BinaryColor::On, 3);
+
+    // let bottom_right = Size::zero() + display.size() - Size::new(0, 0);
+
+    // Rectangle::new(Point::zero(), bottom_right)
+    //     .into_styled(thick_stroke)
+    //     .draw(&mut display)?;
```
Comment (elpiel) [inline examples/cropped_text.rs]: Can you please remove the comments?

File: examples/cropped_text.rs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+pub const DISPLAY_360P: Size = Size::new(480, 360);
+pub const DISPLAY_720P: Size = Size::new(1280, 720);
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /* Algorithms:
+       1. Show a box with an outline
+       2. The box should contain 3 lines of text
+       3. Only the middle line will be fully shown while
+          the others will be partially visible
+    */
```
Comment (elpiel) [inline examples/cropped_text.rs]: There isn't any text written in this example, also I don't see the use of `canvas.crop()` method.
The example should show usage of `crop`

You can use `//` comments instead. `/* */` are really used

File: examples/cropped_text.rs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+use embedded_graphics_simulator::{
+    BinaryColorTheme, OutputSettingsBuilder, SimulatorDisplay, Window,
+};
+
+pub const DISPLAY_240P: Size = Size::new(320, 240);
+pub const DISPLAY_360P: Size = Size::new(480, 360);
+pub const DISPLAY_720P: Size = Size::new(1280, 720);
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /* Algorithms:
```
Comment (elpiel) [inline examples/cropped_text.rs]: instead of `Algorithms` you can write e.g.
We draw:
1. A box with an outline
2. ....
3. ....

**Review by elpiel — CHANGES_REQUESTED** [review]

File: examples/cropped_text.rs (hunk 4 above)
Comment (thecre8tor) [reply examples/cropped_text.rs]: Thanks, Lacho.
I have corrected it.

**Review by elpiel — APPROVED** [review]

**Comment by elpiel** [comment]
:tada: congratulations on your first contribution!

**Pull request merged**